### DISCUSSION TITLE
Affectation de catégorie en cas d'interview

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#2] [Q&A] [answered]
@VeloTerritoires
Dans la méthode (chapitre 3.1.11) , l'introduction indique que

> Lorsque les cyclistes acceptent de répondre à l’enquête approfondie ou à l’interview brève, il est possible de comparer la catégorie renseignée par l’enquêteur et la réponse du cycliste. 

La suite du chapitre indique l'algorithme qui doit être appliqué à partir des réponses du questionnaire mais n'indique pas ce qui doit être fait en cas de réponse à l'interview.

Je suppose que la réponse à l'interview [categorie_breve] remplace la catégorie visuelle [categorie_visuelle_cycliste]?

Dans le fichier "modèle", le champ [categorie_breve] n'est jamais renseigné donc je n'ai pas pu vérifier si ce cas de figure apparaissait souvent

**Comment by SteffMangin** [comment, marked as answer]
La correction par rapport à l'interview brève intervient au paragraphe d'après. Mais elle n'entre pas dans l'algorithme. 
Dans le jeu de données que tu as la [categorie breve] n'a pas été demandée de la même façon que celle proposée dans la méthode. Il y avait juste une confirmation qu'ils étaient itinérants alors que je préconise aujourd'hui de demander la catégorie. C'est pour cela que tu n'as pas d'info dans le champ.

**Reply by JMPivette** [reply]
Effectivement la réponse était un peu plus loin (3.1.12.2). Merci!
